datenguide_python/query_execution.py

```diff
@@ -160,7 +160,7 @@ def _generate_post_json(
         query, variables: Optional[Dict[str, str]] = None
     ) -> Dict[str, str]:
         post_json: Json = dict()
-        post_json["query"] = query.get_graphql_query()
+        post_json["query"] = query.get_graphql_query()[0]
```

Comment:
This makes of course no sense and I would suggest to fix it as part of this pull request directly. I would suggest the following
- merge master into the branch and resolve the conflict
- either you or me does the modifications to the query execution to account for the additional flexibility in the queries.

Reply:
do you want to adapt the QueryExecutioner?
also, the output transformation needs to be adapted as well. (either in the Executioner, only handing one json to the transformer, or in the transformer?)

Reply:
Adaptation done